libs/filameshio/src/MeshReader.cpp

```diff
@@ -159,30 +159,32 @@ static size_t fileSize(int fd) {
 
 namespace filamesh {
 
-MeshReader::Mesh MeshReader::loadMeshFromFile(filament::Engine* engine, const utils::Path& path,
-        MaterialRegistry& materials) {
-
+MeshReader::Mesh MeshReader::loadMeshFromFile(filament::Engine *const engine, const utils::Path& path,
+        MaterialRegistry& materialRegistry) {
     Mesh mesh;
 
-    int fd = open(path.c_str(), O_RDONLY);
+    /* Open the mesh file for reading only. */
+    const int fd = open(path.c_str(), O_RDONLY);
+
+    /* Check if an error occurred when opening the file. */
```

Comment:
Same here, this comment doesn't add anything useful

Reply:
OK. It's a choice.

Reply:
It's useful for someone who is reading the code and doesn't know that -1 is returned when there is an error.
Being useful or not is a relative matter.
You should think about considering the subjectivity of code contributors.